ui/src/app/components/buttons.tsx

```diff
@@ -209,12 +210,28 @@ export const Clickable = ({as = 'div', disabled = false, onClick = null, propaga
   />;
 };
 
-export const Button = ({type = 'primary', style = {}, disabled = false, ...props}) => {
-  return <Clickable
-    // `fp.omit` used to prevent propagation of test IDs to the rendered child component.
-    disabled={disabled} {...fp.omit(['data-test-id'], props)}
-    {...fp.merge(computeStyle(buttonVariants[type], {disabled}), {style})}
-  />;
+export const Button = ({
+                         children,
+                         path='',
+                         type = 'primary',
+                         style = {},
+                         linkStyle={},
+                         disabled = false,
+                         propagateDataTestId = false,
+                         ...props
+                       }) => {
+  // `fp.omit` used to prevent propagation of test IDs to the rendered child component.
+  const childProps = propagateDataTestId ? props : fp.omit(['data-test-id'], props);
+  const computedStyle = fp.merge(computeStyle(buttonVariants[type], {disabled}), {style})
+  return path && !disabled
+      ? <RouteLink path={path} style={buttonVariants[type].linkStyle}>
```

Comment:
why does the enabled version have both a RouteLink and Clickable, but the disabled path has only a Clickable?

Reply:
`Link` does not respect `disabled`. There's a better way to handle this, though, which is to let the prop go through to `RouteLink` and to have _that_ respect `disabled`.